utils/utils.js

```diff
@@ -137,9 +149,11 @@ function handleError(error, analyticsInstance) {
     }
     if (errorMessage && !sampleAdBlockTest) {
       logger.error("[Util] handleError:: ", errorMessage);
+      notifyError(new Error(`[Util] handleError:: ${errorMessage}`));
```

Comment:
Why repeat (in the line above it) the error message twice?

Reply:
done